prusti-interface/src/specs/mod.rs

```diff
@@ -364,7 +365,8 @@ fn get_procedure_spec_ids(def_id: DefId, attrs: &[ast::Attribute]) -> Option<Pro
     );
 
     let pure = has_prusti_attr(attrs, "pure");
-    let trusted = has_prusti_attr(attrs, "trusted");
+    let trusted = has_prusti_attr(attrs, "trusted")
+        || (opt_in_verification() && !has_prusti_attr(attrs, "verified"));
```

Comment:
For consistency; we usually just import `config`.
```suggestion
        || (config::opt_in_verification() && !has_prusti_attr(attrs, "verified"));
```